backends/ref/ceed-ref.c

```diff
@@ -18,9 +18,10 @@
 
 static int CeedInit_Ref(const char *resource, Ceed ceed) {
   int ierr;
-  if (strcmp(resource, "/cpu/self")
+  if (strcmp(resource, "/cpu/self") && strcmp(resource, "/cpu/self/ref")
```

Comment:
I set serial ref to be the default rather than blocked - it is the simplest, which is the purpose of ref.